src/validator/maxfee.validator.ts

```diff
@@ -0,0 +1,10 @@
+import {ExpectedError, ValidationContext, Validator} from 'clime';
+
+export class MaxfeeValidator implements Validator<string> {
```

Comment:
MaxFeeValidator

Reply:
If ``maxFee`` is defined everywhere as  ``number``, why do you validate it as a string?

Reply:
Isn't it possible to accept input of any type? Let me convert number into string.